src/core/Akka.Remote.Tests/EndpointRegistrySpec.cs

```diff
@@ -34,7 +34,7 @@ public void EndpointRegistry_must_be_able_to_register_a_writeable_endpoint_and_p
             var reg = new EndpointRegistry();
             Assert.Null(reg.WritableEndpointWithPolicyFor(address1));
 
-            Assert.Equal(actorA, reg.RegisterWritableEndpoint(address1, actorA));
+            Assert.Equal(actorA, reg.RegisterWritableEndpoint(address1, actorA,null,null));
```

Comment:
The `EndpointRegistry` was modified to include some new data (UIDs for quarantining) and thus the signatures on these methods changed and that needed to be reflected in the specs. 

`EndpointRegistry` is `internal` so it won't have an impact on our users.